src/snowflake/cli/plugins/nativeapp/feature_flags.py

```diff
@@ -0,0 +1,10 @@
+from enum import unique
+
+from snowflake.cli.api.feature_flags import BooleanFlag, FeatureFlagMixin
+
+
+@unique
+class FeatureFlag(FeatureFlagMixin):
+    ENABLE_SETUP_SCRIPT_GENERATION = BooleanFlag(
+        "ENABLE_SETUP_SCRIPT_GENERATION", False
+    )
```

Comment:
Is this PR still WIP? It seems the flag is not used outside tests

Reply:
Yes it is, it's going to be used to gate a feature Bhumika and I are collaborating on this week. We're chopping up the feature into small chunks that will be controlled by this flag.